config/routes.rb

```diff
@@ -57,8 +46,17 @@
     get '/users/me', to: 'users#show'
     get '/users/me/edit', to: 'users#edit'
     patch '/users/me', to: 'users#update'
+
```

Comment:
@shauncarland Thanks for catching this as well as the one above! Space has been removed